spec/sanitized_file_spec.rb

```diff
@@ -20,21 +19,23 @@
     FileUtils.rm_rf(public_path)
   end
 
-  describe '#empty?' do
-
+  describe "#empty?" do
     it "should be empty for nil" do
```

Comment:
I feel these examples would be better written more like:

```ruby
describe "#empty?" do
  let(:sanitized_file) { CarrierWave::SanitizedFile.new(file) }

  subject {  sanitized_file.empty? }

  context "when SanitizedFile initialized with nil"
    let(:file) { nil }

    specify { expect(subject).to be_true }
  end

  context "when SanitizedFile initialized with an empty string"
    let(:file) { "" }

    specify { expect(subject).to be_true }
  end

  # ...  
end
```

The `specify { expect(subject).to ...` can also be written as `it {  should ...`. I also have a gem (`rspec-expect_it`) that allows writing `specify { expect_it.to ...`.